code/modules/mob/living/carbon/xenomorph/Evolution.dm

```diff
@@ -149,13 +149,10 @@
 						to_chat(src, "<span class='warning'>You shouldn't see this. If you do, bug repot it! (Error XE01).</span>")
 
 						continue
-				if(M.client && M.ckey)
-					totalXenos++
-
-		if(tier == 1 && ((tierB + tierC) / max(totalXenos, 1))> 0.5)
+		if(tier == 1 && ( (tierB + tierC) > tierA ) )
 			to_chat(src, "<span class='warning'>The hive cannot support another Tier 2, wait for either more aliens to be born or someone to die.</span>")
 			return
-		else if(tier == 2 && (tierC / max(totalXenos, 1))> 0.25)
+		else if(tier == 2 && ( (tierC * 3) > (tierA + tierB) ) )
```

Comment:
the 3 is a magic number thats used below, should be a define to avoid them becoming different values